pandas/tests/io/test_pytables.py

```diff
@@ -5264,6 +5264,19 @@ def test_fspath(self):
             with pd.HDFStore(path) as store:
                 assert os.fspath(store) == str(path)
 
+    def test_read_py2_hdf_file_in_py3(self):
+        # GH 16781
+
+        # tests reading a PeriodIndex DataFrame written in Python2 in Python3
```

Comment:
even better if u can name this file similar to how we name the files in legacy_pickle dirs